### ISSUE TITLE
getClosestVueParent in <md-table> bug

### ISSUE THREAD

**Issue by Andkoo** [vuematerial/vue-material#661]
### Steps to reproduce
• I am using `<md-table>` in my custom component without wrapper `<md-table-card>`.
• The parent component is `<md-card>` with `<md-tabs>` in it. Every `<md-tab>` contains one custom component with `<md-table>` in it

### Which browser?
electron 1.6.3

### What is actually happening?
Everything works, but I get 2 errors:
• [Vue warn]: Error in mounted hook: (found in MdTable)
• TypeError: Cannot read property 'contains' of undefined

I have found out that this is caused by 
```
mounted() {
  this.parentCard = getClosestVueParent(this.$parent, 'md-table-card');

  if (this.parentCard) {
    this.parentCard.tableInstance = this;
  }
}
```

in `MdTable.vue` and can be fixed by wrapping the `<md-table>` in `<md-table-card>` which I don't find a good solution as long as it's parent is `<md-card>`.

**Comment by pablohpsilva** [comment]
Hello @Andkoo !
You probably got your workaround working, but I'm sure I'll fix any issue with mdTable ASAP.
Thank you for opening this issue. You rock.

**Comment by pablohpsilva** [comment]
Probably fixed on #787
